railties/lib/rails/commands/app/update_command.rb

```diff
@@ -87,6 +89,13 @@ def asset_pipeline
               nil
             end
           end
+
+          def skip_gem?(gem_name)
+            require gem_name
```

Comment:
I wonder if `gem gem_name` works here. Requiring rubocop takes quite a while, 0.7 seconds on my machine and that would be quite a bit faster.

Reply:
Yes good catch, I also get some time saved (the printed symbols are just to make sure the code runs):
```console
etiennebarrie/rails-app (bin-brakeman-fixed-added $) $ time bin/rails app:update:bin --force                                                                                                                                                                                    
:require
:require
       exist  bin
   identical  bin/brakeman
   identical  bin/rails
   identical  bin/rake
   identical  bin/rubocop
   identical  bin/setup

real	0m0.725s
user	0m0.471s
sys	0m0.247s
etiennebarrie/rails-app (bin-brakeman-fixed-added $) $ time bin/rails app:update:bin --force                                                                                                                                                                                    
:gem
:gem
       exist  bin
   identical  bin/brakeman
   identical  bin/rails
   identical  bin/rake
   identical  bin/rubocop
   identical  bin/setup

real	0m0.424s
user	0m0.261s
sys	0m0.153s
```